test/scripts/deriveTestIds.ts

```diff
@@ -19,106 +19,108 @@ import yargs from "yargs";
+            // If it's Z, wrap around to A
+            prefix = String.fromCharCode(prefix.charCodeAt(0) + 1) + "A";
+        } else {
+            prefix = prefix[0] + String.fromCharCode(charCode + 1);
+        }
     }
-  }
 
-  usedPrefixes.add(prefix);
-  return prefix;
+    usedPrefixes.add(prefix);
+    return prefix;
 }
 
 function generateId(directory: string, rootDir: string, prefix: string): void {
-  let contents = fs.readdirSync(directory);
-
-  contents.sort((a, b) => {
-    const aIsDir = fs.statSync(path.join(directory, a)).isDirectory();
-    const bIsDir = fs.statSync(path.join(directory, b)).isDirectory();
-
-    if (aIsDir && !bIsDir) return -1;
-    if (!aIsDir && bIsDir) return 1;
-    return customFileSort(a, b);
-  });
-
-  let fileCount = 1;
-  let subDirCount = 1;
-
-  for (const item of contents) {
-    const fullPath = path.join(directory, item);
-
-    if (fs.statSync(fullPath).isDirectory()) {
-      const subDirPrefix = ("0" + subDirCount).slice(-2);
-      generateId(fullPath, rootDir, prefix + subDirPrefix);
-      subDirCount++;
-    } else {
-      const fileContent = fs.readFileSync(fullPath, "utf-8");
-      if (fileContent.includes("describeSuite")) {
-        const newId = prefix + ("0" + fileCount).slice(-2);
-        const updatedContent = fileContent.replace(
-          /(describeSuite\s*?\(\s*?\{\s*?id\s*?:\s*?['"])[^'"]+(['"])/,
-          `$1${newId}$2`
-        );
-        fs.writeFileSync(fullPath, updatedContent);
-      }
-      fileCount++;
+    const contents = fs.readdirSync(directory);
+
+    contents.sort((a, b) => {
+        const aIsDir = fs.statSync(path.join(directory, a)).isDirectory();
+        const bIsDir = fs.statSync(path.join(directory, b)).isDirectory();
+
+        if (aIsDir && !bIsDir) return -1;
+        if (!aIsDir && bIsDir) return 1;
+        return customFileSort(a, b);
+    });
+
+    let fileCount = 1;
+    let subDirCount = 1;
+
+    for (const item of contents) {
+        const fullPath = path.join(directory, item);
+
+        if (fs.statSync(fullPath).isDirectory()) {
+            const subDirPrefix = ("0" + subDirCount).slice(-2);
+            generateId(fullPath, rootDir, prefix + subDirPrefix);
+            subDirCount++;
+        } else {
+            const fileContent = fs.readFileSync(fullPath, "utf-8");
+            if (fileContent.includes("describeSuite")) {
+                const newId = prefix + ("0" + fileCount).slice(-2);
+                const updatedContent = fileContent.replace(
+                    /(describeSuite\s*?\(\s*?\{\s*?id\s*?:\s*?['"])[^'"]+(['"])/,
+                    `$1${newId}$2`
+                );
+                fs.writeFileSync(fullPath, updatedContent);
+            }
+            fileCount++;
+        }
     }
-  }
 }
 
 function hasSpecialCharacters(filename: string): boolean {
-  return /[ \t!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]+/.test(filename);
+    return /[ \t!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]+/.test(filename);
```

Comment:
This regex has changed, was this prettier or was it you?

Reply:
ESLint told me that the escaping was redundant in this example